CHANGELOG.md

```diff
@@ -10,12 +10,16 @@
 * Added `ViewManagerModel.preserveUnsavedChanges` flag to opt-out of that behaviour.
 * Added `PersistOptions.settleTime` to configure time to wait for state to settle before persisting.
 * Support for gridcolumn level `onCellClicked` events.
+* Chart context menu is now fully customizable.
```

Comment:
We could mention in the context of "general improvements to menu api"  and also mention new MenuContext object now sent generally to both handlers as second argument.